modules/react/common/stories/examples/CreateModelHook.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  createContainer,
+  createModelHook,
+  createSubcomponent,
+  createElemPropsHook,
+  useUniqueId,
+} from '@workday/canvas-kit-react/common';
+import {PrimaryButton, PrimaryButtonProps} from '@workday/canvas-kit-react/button';
+import {Box, BoxProps} from '@workday/canvas-kit-react/layout';
+
+export type Visibility = 'hidden' | 'visible';
+
+// First we define a model using out createModelHook
+const useDisclosureModel = createModelHook({
+  // This becomes the default values on the model
+  defaultConfig: {
+    /** ID reference of the list. Children ids can be derived from this id */
+    id: '',
+    /**
+     * The initial visibility of the disclosed content
+     * @default 'hidden'
+     */
+    initialVisibility: 'hidden' as Visibility,
+  },
+})(config => {
+  const id = useUniqueId(config.id);
+  const [visibility, setVisibility] = React.useState(config.initialVisibility || 'hidden');
```

Comment:
This was a mistake during conversion. The `defaultConfig` sets defaults. Also, the `@default` is no longer needed. The `|| hidden` is also weakening the type information and will be removed during the improved documentation work.

```suggestion
  const [visibility, setVisibility] = React.useState(config.initialVisibility);
```